src/_playground/App.scss

```diff
@@ -571,3 +571,134 @@ a.toc-link {
         }
     }
 }
+
+
```

Comment:
we can use separate file here rather than using app.scss as a dumping ground.

seems like we should break this up. it also might allow use to better segment out the CSS that is intended to be temporary

Reply:
Good point.  I pushed a change to do this.